src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIProjectSubscriptionService.java

```diff
@@ -0,0 +1,76 @@
+package ca.corefacility.bioinformatics.irida.ria.web.services;
+
+import ca.corefacility.bioinformatics.irida.model.enums.ProjectRole;
+import ca.corefacility.bioinformatics.irida.model.joins.impl.ProjectUserJoin;
+import ca.corefacility.bioinformatics.irida.model.project.Project;
+import ca.corefacility.bioinformatics.irida.model.subscription.ProjectSubscription;
+import ca.corefacility.bioinformatics.irida.model.user.User;
+import ca.corefacility.bioinformatics.irida.model.user.group.UserGroupProjectJoin;
+import ca.corefacility.bioinformatics.irida.ria.web.models.tables.TableRequest;
+import ca.corefacility.bioinformatics.irida.ria.web.models.tables.TableResponse;
+import ca.corefacility.bioinformatics.irida.ria.web.users.dto.UserProjectDetailsModel;
+import ca.corefacility.bioinformatics.irida.service.ProjectService;
+import ca.corefacility.bioinformatics.irida.service.ProjectSubscriptionService;
+import ca.corefacility.bioinformatics.irida.service.user.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.stereotype.Component;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * A utility class for formatting responses for the project subscriptions page UI.
+ */
+@Component
+public class UIProjectSubscriptionService {
```

Comment:
Can you fix the formatting of this file, it should be indented with tabs and not spaces. Might just need to update your ide to set tabs as default for .java files.

Reply:
Please see 242aeb0522f0fe6211596050c7f3a2dcc6ef7b32.